website/templates/metadata/register_1.mako

```diff
@@ -47,6 +47,6 @@
     window.contextVars.node.children = ${[str(each) for each in children_ids]};
     window.contextVars.regTemplate = ${json.dumps(template_name or '')};
     window.contextVars.regSchema = ${schema};
-    window.contextVars.regPayload = ${json.dumps(payload)};
+    window.contextVars.regPayload = $.parseJSON(${json.dumps(payload)});
```

Comment:
This shouldn't be necessary. Could you instead remove the JSONification of the already JSONified `payload` here: https://github.com/CenterForOpenScience/osf.io/blob/145ed8b009f0a1f36cd0fdab4c6850b1279af986/website/project/views/register.py#L69 ?